detect_secrets/plugins/common/filters.py

```diff
@@ -97,3 +78,79 @@ def _is_sequential_string(secret, *args):
             return True
 
     return False
+
+
+ALL_FALSE_POSITIVE_HEURISTICS = (
+    is_found_with_aho_corasick,
+    is_sequential_string,
+)
+
+
+# NOTE: this doesn't handle key-values on a line properly.
+# NOTE: words that end in "id" will be treated as ids
+_ID_DETECTOR_REGEX = re.compile(r'[iI][dD][^A-Za-z0-9]')
```

Comment:
We might be able to do `_id`, we'll see what the data says though.

Reply:
I guess it depends on whether we want to ignore keys like `BusinessId`. I think at Yelp this isn't likely but it's probably more likely in `camelCase` language repos.

Reply:
That's a good point, we do have a lot of python biases.